### ISSUE TITLE
Publish releases

### ISSUE THREAD

**Issue by VojtechVitek** [kardianos/govendor#119]
It's quite hard to follow new features coming to the master branch. Do you think you could tag & release certain features so we can follow the project more easily?

**Comment by kardianos** [comment]
On my TODO list. Right now I'm still in v0 effectively.

However, once I've got the sync, fetch, and related changes stabilized, I do plan to both tag releases and publish binaries.

## Roadmap:

 * v1.0 after known bugs are fixed and people have tried it out for a bit.
 * v1.1 after govendor handles dependent vendor file with prompts for developer


v1.0 will likely come mid next-week.

Thanks for adding an issue to track this.

**Comment by VojtechVitek** [comment]
Thank you.

**Comment by kardianos** [comment]
I've began doing this.
